### ISSUE TITLE
Demo not working in FireFox

### ISSUE THREAD

**Issue by xoryouyou** [bjnortier/hershey#1]
Mousewheel and event.offset were not working in FF.
So FTFY ;)

```
--- demo/js/src/trackball.js	(revision d28a196c396908572085d1252297542c6a249600)
+++ demo/js/src/trackball.js	(revision dcf7877383f7257b9d2120ed8c6ec4ee9ec910cf)
@@ -1,4 +1,7 @@
 function eventToPosition(event) {
+  event.offsetX = (event.offsetX === undefined) ? event.layerX : event.offsetX;
+  event.offsetY = (event.offsetY === undefined) ? event.layerY : event.offsetY;
+
   return {
     x: event.offsetX,
     y: event.offsetY,
@@ -103,10 +106,12 @@
     viewport.container.addEventListener('mouseup', that.mouseup, false);
     viewport.container.addEventListener('mouseout', that.mouseup, false);
     viewport.container.addEventListener('mousewheel', that.mousewheel, false);
+    viewport.container.addEventListener('DOMMouseScroll', that.mousewheel, false);
   });
 
   animate();
 
 };
+
```

**Comment by bjnortier** [comment]
woop!

**Comment by bjnortier** [comment]
released 1.0.2
